client/src/components/Modals/EditableTags/EditableTag.tsx

```diff
@@ -56,7 +58,7 @@ const EditableTagModal: FC<EditableTagModalProps> = ({
     }));
 
     setClonedTags([...clone]);
-
+    onFetch();
     // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
This is what I meant can be removed.

Reply:
bruh. why does it choose the wrong line. line 62. the eslint-disable.

Reply:
Ye but removing it brings back the lint warning `React Hook useEffect has missing dependencies: 'allTags' and 'onFetch'. Either include them or remove the dependency array. If 'onFetch' changes too often, find the parent component that defines it and wrap that definition in useCallback`

Reply:
These should have been addressed before merging.